### ISSUE TITLE
Have code editor side-by-side with mentor discussion

### ISSUE THREAD

**Issue by Steffan153** [exercism/exercism#5943]
(Feature request.)

Often when you have several suggestions, it would be nice if you could have the code editor side-by-side with the mentor discussion. It's kind of annoying for me to have to switch between tabs constantly, and always open both.

**Comment by iHiD** [comment]
Yeah. We're planning do have a fully fleshed out sync discussion flow as of Exercism Teams early next year. 

I feel it's important that the normal mentor discussion remains async, but we could certainly put the comments into editor in a readonly manner to start with (having read-write would be really confusing, as the student could be discussing WIP code that the mentor couldn't see).
